OpenRA.Mods.Common/Activities/Repair.cs

```diff
@@ -68,7 +68,7 @@ public override Activity Tick(Actor self)
 			if (host.Type == TargetType.Invalid || health == null)
 				return NextActivity;
 
-			if (closeEnough.LengthSquared > 0 && !host.IsInRange(self.CenterPosition, closeEnough))
+			if (closeEnough.Length != 0 && !host.IsInRange(self.CenterPosition, closeEnough))
```

Comment:
In Repair.cs, closeEnough is WDist class, who already has Length inside the class and computing LengthSquared is an extra work.